lenses_python/Policy.py

```diff
@@ -0,0 +1,33 @@
+from requests import *
```

Comment:
Try to avoid this import with asterisk. Import only what you are using in this piece of code from requests package

Reply:
ok

Reply:
This will be fixed on 2.2 with a new branch as it requires to re-write many aspects of the code